Rakefile

```diff
@@ -84,7 +84,7 @@ namespace :docs do
           end
 
         types = if method.tag(:return)&.types
-          " → [#{method.tag(:return).types.join(',')}]"
+                  " → [#{method.tag(:return).types.join(',')}]"
```

Comment:
This seems like it's indented too far. I generally have a preference for consistent indentation instead of trying to align to the right of the `if` statement above.

Thoughts?

Reply:
I think this may be the relevant cop for this behavior? https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilineassignmentlayout